project/Dependencies.scala

```diff
@@ -196,15 +196,19 @@ object Dependencies {
     // FS2
     val decline          = "com.monovore"                     %% "decline"                               % V.decline
     val fs2PubSub        = "com.permutive"                    %% "fs2-google-pubsub-grpc"                % V.fs2PubSub
-    val fs2Aws           = ("io.laserdisc"                    %% "fs2-aws"                               % V.fs2Aws).exclude("com.amazonaws", "amazon-kinesis-producer")
+    val fs2Aws           = ("io.laserdisc"                    %% "fs2-aws"                               % V.fs2Aws)
+                           .exclude("com.amazonaws", "amazon-kinesis-producer")
+                           .exclude("software.amazon.kinesis", "amazon-kinesis-client")
```

Comment:
Don't the versions of this `amazon-kinesis-client` always get overridden by the ones of `kinesisClient2` ?

Reply:
Yes it gets overridden, but this is to enforce that the `software.amazon.glue` packages are properly excluded [further down](https://github.com/snowplow/enrich/blob/3.5.1-rc2/project/Dependencies.scala#L210-L211).  Without this line, I still end up with `software.amazon.glue` in the final assembly.

Reply:
Oh I see !